code/modules/uplink/uplink_items.dm

```diff
@@ -1885,6 +1922,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 2
 	illegal_tech = FALSE
 
+/datum/uplink_item/badass/clownopclumsinessinjector //clowns can buy this too, but it's in the role-restricted items section for them
```

Comment:
Can you make it a child of `/datum/uplink_item/role_restricted/clumsinessinjector` to avoid the duplicate code?

Reply:
no, because then it would go into the role restricted section, right?

Reply:
Makes sense.